authentication.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const jwt = require('jsonwebtoken');
+require('dotenv').config()
+
+const secretKey = process.env.KEY;
+
+const authenticated = express.Router(); 
+
+authenticated.use(function(req, res, next) {
+  let token = req.headers['auth'];
+  if (token) {
+    jwt.verify(token, secretKey, function(err, decoded) {      
+      if (err) {
+        return res.status(401).json({ "status": "error", "message": "Authentication required" });    
+      } else {
+        next();
+      }
+    });
+  } else {
+    return res.status(401).json({ "status": "error", "message": "Authentication required" });
```

Comment:
`{ "status": "error", "message": "Authentication required" }` could be in a variable, since it's used in line 14 also.